site/cypress/e2e/Cypress-Gradeable/rainbow_grading.spec.js

```diff
@@ -22,7 +22,7 @@ describe('Test Rainbow Grading', () => {
         cy.get('[data-testid="display-benchmarks-stddev"]').check();
         cy.get('[data-testid="display-benchmarks-perfect"]').check();
         cy.get('[data-testid="save-status-button"]').click();
-        cy.get('[data-testid="save-status"]', { timeout: 500000 }).should('contain', 'Rainbow grades successfully generated!');
+        cy.get('[data-testid="save-status"]', { timeout: 15000 }).should('contain', 'Rainbow grades successfully generated!');
```

Comment:
Is it possible to do it without any explicit timeout?

Reply:
We can not move to the next segment of the tests without generating the summary and it does take some time. I am not sure if there exists another way to generate the rainbow grades.